opentelemetry/proto/resource/v1/resource.proto

```diff
@@ -16,12 +16,20 @@ syntax = "proto3";
 
 package opentelemetry.proto.resource.v1;
 
+import "opentelemetry/proto/common/v1/common.proto";
+
 option java_multiple_files = true;
 option java_package = "io.opentelemetry.proto.resource.v1";
 option java_outer_classname = "ResourceProto";
 
 // Resource information.
 message Resource {
-  // Set of labels that describe the resource.
-  map<string,string> labels = 1;
+  // labels is a collection of attributes that describe the resource. See OpenTelemetry
+  // specification semantic conventions for standardized label names:
+  // https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-resource-semantic-conventions.md
+  repeated opentelemetry.proto.common.v1.AttributeKeyValue labels = 1;
+
+  // dropped_labels_count is the number of dropped labels. If the value is 0, then
+  // no labels were dropped.
+  int32 dropped_labels_count = 2;
```

Comment:
I don't think we have yet define the capability to drop labels from resource.

Reply:
Intermediary nodes can drop labels to save bandwidth, etc. It does not necessarily have to be part of OpenTelemetry API or SDK.